crates/runtime/src/env/default/serialize.rs

```diff
@@ -23,46 +17,21 @@ impl AccountSerializer for DefaultAccountSerializer {
     fn serialize(account: &ExtAccount) -> Vec<u8> {
         let mut w = Vec::new();
 
-        encode_template(account, &mut w);
-        encode_name(account, &mut w);
-        encode_spawner(account, &mut w);
+        account.template_addr().0.encode(&mut w);
+        account.name().to_string().encode(&mut w);
+        account.spawner().0.encode(&mut w);
 
         w
     }
 }
 
-fn encode_template(account: &ExtAccount, w: &mut Vec<u8>) {
-    let addr = account.template_addr();
-    w.write_template_addr(addr);
-}
-
-fn encode_name(account: &ExtAccount, w: &mut Vec<u8>) {
-    w.write_string(account.name());
-}
-
-fn encode_spawner(account: &ExtAccount, w: &mut Vec<u8>) {
-    let spawner = account.spawner();
-    w.write_address(spawner);
-}
-
 impl AccountDeserializer for DefaultAccountDeserializer {
     fn deserialize(bytes: &[u8]) -> Option<ExtAccount> {
-        let mut cursor = Cursor::new(bytes);
+        let mut reader = Cursor::new(bytes);
```

Comment:
I would keep the name `cursor` here (since the type is `Cursor` as well)

Reply:
:+1: